server.go

```diff
@@ -17,12 +18,16 @@ import (
 	"github.com/getsentry/raven-go"
 	"github.com/zenazn/goji/bind"
 	"github.com/zenazn/goji/graceful"
+
+	"github.com/pkg/profile"
 )
 
 // VERSION stores the current veneur version.
 // It must be a var so it can be set at link time.
 var VERSION = "dirty"
 
+var profileStartOnce = sync.Once{}
```

Comment:
curious, do you feel that this Once is necessary? if someone adds a profile.Start somewhere else in the code, I don't think the Once here will protect them (because they'll probably forget to use it)